### ISSUE TITLE
Ordinal not in range

### ISSUE THREAD

**Issue by nicokoch** [mps-youtube/mps-youtube#351]
I just installed this program and it doesn't work correctly. I searched by typing `/eminem`, and this is the error output:
```
Traceback (most recent call last):
  File "./mpsyt", line 3, in <module>
    mps_youtube.main.main()
  File "/home/kokxx/dev/github/mps-youtube/mps_youtube/main.py", line 3982, in main
    screen_update()
  File "/home/kokxx/dev/github/mps-youtube/mps_youtube/main.py", line 967, in screen_update
    xprint(g.content)
  File "/home/kokxx/dev/github/mps-youtube/mps_youtube/util.py", line 93, in xprint
    print(xenc(stuff), end=end)
UnicodeEncodeError: 'ascii' codec can't encode character '\xf1' in position 1832: ordinal not in range(128)
WARNING:root:Unknown itag: 13
```

**Comment by ids1024** [comment]
This may have been fixed in the develop branch. Try running `pip3 install --upgrade https://github.com/mps-youtube/mps-youtube/archive/develop.zip` to install that and try again.

**Comment by nicokoch** [comment]
I am already running the newest git version from the develop branch.

I have found out running it with `PYTHONIOENCODING=utf_8 mpsyt` fixes the problem.

**Comment by ids1024** [comment]
Ah, that's useful. I can use `PYTHONIOENCODING=ascii mpsyt` to reproduce it.

This should be fixed in commit d884326fa5826f722da200bff81efbe52c7ed85e.

**Comment by nicokoch** [comment]
Yep, that fixed it. Thanks!
